pkgs/development/python-modules/scikit-network/default.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+
+  cython,
+  setuptools,
+  numpy,
+  scipy,
+  pytest-runner,
+  pytest,
+  nose,
+  pluggy,
+}:
+buildPythonPackage rec {
+  pname = "scikit-network";
+  version = "0.33.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-9/fMAutCZ7Fqk4P2RiYVLlm6Q11td7GjVmW2/BvPVy0=";
+  };
+
+  # can be removed if https://github.com/sknetwork-team/scikit-network/pull/581 is merged
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-warn "__builtins__.__NUMPY_SETUP__ = False" "__builtins__['__NUMPY_SETUP__'] = False"
+  '';
+
+  build-system = [
+    cython
+    setuptools
+    numpy
+    scipy
+    pytest-runner
+  ];
+
+  dependencies = [
+    numpy
+    scipy
+  ];
+
+  nativeCheckInputs = [
+    pytest
+    nose
+    pluggy
+  ];
+
+  pythonImportscheck = [ "sknetwork" ];
+
+  meta = with lib; {
+    description = "Free software library in Python for machine learning on graphs";
+    changelog = "https://pypi.org/project/scikit-network/${version}/";
+    homepage = "https://scikit-network.readthedocs.io";
+    license = licenses.bsd;
```

Invalid license, based on their [LICENSE](https://github.com/sknetwork-team/scikit-network/blob/master/LICENSE) file it should be the [BSD 3-Clause](https://opensource.org/license/BSD-3-clause) license.

```suggestion
    license = licenses.bsd3;
```